keanu-python/tests/test_vertex.py

```diff
@@ -0,0 +1,88 @@
+import keanu as kn
+import numpy as np
+import pytest
+
+
+@pytest.fixture
+def jvm_view():
+    from py4j.java_gateway import java_import
+    jvm_view = kn.KeanuContext().jvm_view()
+    java_import(jvm_view, "io.improbable.keanu.vertices.dbl.probabilistic.GaussianVertex")
+    return jvm_view
+
+
+def test_can_pass_scalar_to_vertex(jvm_view):
+    gaussian = kn.Vertex(jvm_view.GaussianVertex, (0., 1.))
+    sample = gaussian.sample()
+
+    assert sample.isScalar()
+
+
+def test_can_pass_ndarray_to_vertex(jvm_view):
+    gaussian = kn.Vertex(jvm_view.GaussianVertex, (np.array([[0.1, 0.4]]), np.array([[0.4, 0.5]])))
+    sample = gaussian.sample()
+
+    shape = sample.getShape()
+    assert sample.getRank() == 2
+    assert shape[0] == 1
+    assert shape[1] == 2
+
+
+def test_can_pass_java_object_to_vertex(jvm_view):
```

Comment:
I think my earlier comment got eaten up by newer commits - 
I think this should be named `test_can_pass_vertex_to_vertex ` because you can't pass it any old Java object.

Reply:
I commented that nothing on the python side is preventing the user from passing any old Java object - but agreed. It'll still throw an exception from the java side.